website/docs/modules/gather_batch_evidence.md

```diff
@@ -34,148 +35,86 @@ stateDiagram
 ```
 
 ## Inputs
-This workflow takes as input the read counts, BAF, PE, SD, SR, and per-caller VCF files 
-produced in the GatherSampleEvidence workflow, and contig ploidy and gCNV models from 
-the TrainGCNV workflow.
-The following is the list of the inputs the GatherBatchEvidence workflow takes.
 
+:::info
+All array inputs of sample data must match in order. For example, the order of the `samples` array should match that of
+`counts`, `PE_files`, etc.
+:::
 
 #### `batch`
-An identifier for the batch.
-
+An identifier for the batch; may only be alphanumeric with underscores.
 
 #### `samples`
-Sets the list of sample IDs. 
+Sample IDs. Must match the sample IDs used in [GatherSampleEvidence](./gse#sample_id) unless `rename_samples` is enabled, in 
+which case sample IDs will be overwritten. See [sample ID requirements](/docs/gs/inputs#sampleids) for specifications 
+of allowable sample IDs.
 
+#### `ped_file`
+Family structures and sex assignments determined in [EvidenceQC](./eqc). See [PED file format](/docs/gs/inputs#ped-format).
 
 #### `counts`
-Set to the [`GatherSampleEvidence.coverage_counts`](./gse#coverage-counts) output.
-
-
-#### Raw calls
-
-The following inputs set the per-caller raw SV calls, and should be set 
-if the caller was run in the [`GatherSampleEvidence`](./gse) workflow.
-You may set each of the following inputs to the linked output from 
-the GatherSampleEvidence workflow.
-
-
-- `manta_vcfs`: [`GatherSampleEvidence.manta_vcf`](./gse#manta-vcf);
-- `melt_vcfs`: [`GatherSampleEvidence.melt_vcf`](./gse#melt-vcf);
-- `scramble_vcfs`: [`GatherSampleEvidence.scramble_vcf`](./gse#scramble-vcf);
-- `wham_vcfs`: [`GatherSampleEvidence.wham_vcf`](./gse#wham-vcf).
+Binned read count files (`*.rd.txt.gz`) generated in [GatherSampleEvidence](./gse#coverage-counts).
 
 #### `PE_files`
-Set to the [`GatherSampleEvidence.pesr_disc`](./gse#pesr-disc) output.
+Discordant pair evidence files (`*.pe.txt.gz`) generated in [GatherSampleEvidence](./gse#pesr-disc).
 
 #### `SR_files`
-Set to the [`GatherSampleEvidence.pesr_split`](./gse#pesr-split)
-
+Split read evidence files (`*.sr.txt.gz`) generated in [GatherSampleEvidence](./gse#pesr-split).
 
 #### `SD_files`
-Set to the [`GatherSampleEvidence.pesr_sd`](./gse#pesr-sd)
-
-
-#### `matrix_qc_distance`
-You may refer to [this file](https://github.com/broadinstitute/gatk-sv/blob/main/inputs/templates/terra_workspaces/cohort_mode/workflow_configurations/GatherBatchEvidence.json.tmpl)
-for an example value. 
-
-
-#### `min_svsize`
-Sets the minimum size of SVs to include.
-
-
-#### `ped_file`
-A pedigree file describing the familial relationshipts between the samples in the cohort.
-Please refer to [this section](./#ped_file) for details. 
+Site depth files (`*.sd.txt.gz`) generated in [GatherSampleEvidence](./gse#pesr-sd).
 
+#### `*_vcfs`
```

Comment:
You may override the section ids as `{#vcfs}` to simplify referencing it (you would not need to escape `_` and avoid asterisk).

Reply:
I haven't been totally consistent with this, but for most pages I left the identifiers alone to avoid verbosity.